tests/pipeline/resources/streams-bootstrap/defaults.yaml

```diff
@@ -1,6 +1,7 @@
 streams-bootstrap:
   version: "3.0.3"
   values:
+    imagePullPolicy: IfNotPresent
```

Comment:
are you sure about this change? this deviates from the default defined in streams-bootstrap.

Reply:
I added the imagePullPolicy here only for testing. The problem was that when we called `kpops manifest ...` the values of the enums were not dumped as String. Here is the error:
```
yaml.representer.RepresenterError: ('cannot represent an object', <ImagePullPolicy.IF_NOT_PRESENT: 'IfNotPresent'>)
```

Reply:
gotcha! if you revert the change I'll approve :)